README.md

```diff
@@ -6,11 +6,79 @@
 ## Summary
 This protected microservice provides a backend for Making Tax Digital for Businesses (MTDfB) VAT frontend services to retrieve VAT Return Obligation details for a user enrolled for MTD VAT.
 
+## Endpoint Definitions (APIs)
+
+**Method**: GET
+
+**Path**: /enterprise/obligation-data/:vatNumber
+
+Where:
+
+* **:vatNumber** is a valid VRN, for example: "999999999"
+
+**Query Parameters**:
+
+|Query Parameter|Mandatory|Description|Format/Valid Values|    
+|-|-|-|-|    
+|`from`|**false**|Used to filter the response to only include items from this date|YYYY-MM-DD|    
+|`to`|**false**|Used to filter the response to only include items before this date|YYYY-MM-DD|    
+|`status`|**false**|Used to filter the response to only include items that have a specific status|`String`|
+
+
+#### Success Response
+
+##### VatObligations object
+|Data Item|Type|Mandatory|    
+|-|-|-|    
+|obligations|`Array[VatObligation]`|**true**|
+
+##### VatObligation object
+|Data Item|Type|Mandatory|    
+|-|-|-|    
+|obligations|`Array[ObligationDetail]`|**true**|
+
+##### ObligationDetail object
+|Data Item|Type|Mandatory|    
+|-|-|-|    
+|status|`String`|**true**|
+|start|`String`|**true**|
+|end|`String`|**true**|
+|received|`String`|**false**|
+|due|`String`|**true**|
+|periodKey|`String`|**true**|
+
+#### Example
+
+Status: OK (200)
+
+Response Body:
+```
+{    
+    "VatObligations": [      
```

Comment:
This sample response and some of the tables above need tweaking. This doesn't look like the correct format of data this endpoint returns. To see the data you could send a curl request to the endpoint or put some temporary logging in place. Alternatively you could look at the IT tests in the frontends that call this endpoint.